EmployeeLeaveTracking/EmployeeLeaveTracking.Services/Services/LeaveRequestService.cs

```diff
@@ -363,7 +398,7 @@ public async Task<int> UpdateLeaveRequestStatus(int id, int statusId)
             _dbContext.Entry(leaveRequest).State = EntityState.Modified;
             await _dbContext.SaveChangesAsync();
 
-            if (statusId == 2 || statusId == 3) 
+            if (statusId == 2 || statusId == 3)
```

Comment:
what is 2 | 3 here, why are we not making use of Enums

Reply:
Had Already added the StatusType Table in Database with Data Seeding Concept